docs/release-notes/1.2.0/release-1.2.0.md

```diff
@@ -0,0 +1,70 @@
+
+    This package makes it easier to use [ML.NET](dot.net/ml) with app models that support Microsoft.Extensions - i.e. ASP.NET and Azure Functions.
+
+    Specifically it contains functionality for:
+    - Dependency Injection
+    - Pooling PredictionEngines
+    - Reloading models when the file or URI has changed
+    - Hooking ML.NET logging to Microsoft.Extensions.Logging
+
+## **Bug Fixes**
+### Serious
+- **Time series Sequential Transform needs to have a binding mechanism:** This bug made it impossible to use time series in NimbusML. ([#3875](https://github.com/dotnet/machinelearning/pull/3875))
+
+- **build errors resulting from upgrading to VS2019 compilers:** The default CMAKE_C_FLAG for debug configuration sets /ZI to generate a PDB capable of edit and continue. In the new compilers, this is incompatible with /guard:cf which we set for security reasons. ([#3894](https://github.com/dotnet/machinelearning/pull/3894)) 
+
+- **LightGBM Evaluation metric parameters:** In LightGbm EvaluateMetricType where if a user specified EvaluateMetricType.Default, the metric would not get added to the options Dictionary, and LightGbmWrappedTraining would throw because of that. ([#3815](https://github.com/dotnet/machinelearning/pull/3815))
+
+- **Change default EvaluationMetric for LightGbm:** In [ML.NET](dot.net/ml), the default EvaluationMetric for LightGbm is set to EvaluateMetricType.Error for multiclass, EvaluationMetricType.LogLoss for binary, and so on. This leads to inconsistent behavior from the user's perspective. ([#3859](https://github.com/dotnet/machinelearning/pull/3859))
+### Other
+- CustomGains should allow multiple values in argument attribute. ([#3854](https://github.com/dotnet/machinelearning/pull/3854))
+
+## **Breaking Changes**
+None
+
+## **Enhancements**
+- Fixes the Hardcoded Sigmoid value from -.5 to the value specified during training. ([#3850](https://github.com/dotnet/machinelearning/pull/3850))
+- Fix TextLoader constructor and add exception message. ([#3788](https://github.com/dotnet/machinelearning/pull/3788))
+- Introduce the `FixZero` argument to the LogMeanVariance normalizer. ([#3916](https://github.com/dotnet/machinelearning/pull/3916))
+- Ensembles trainer now work with ITrainerEstimators instead of ITrainers, ([#3796](https://github.com/dotnet/machinelearning/pull/3796)) 
+- LightGBM Unbalanced Data Argument. ([#3925](https://github.com/dotnet/machinelearning/pull/3925))
+- Tree based trainers implement ICanGetSummaryAsIDataView. ([#3892](https://github.com/dotnet/machinelearning/pull/3892))
+
+## **Documentation and Samples**
+- Samples for applying ONNX model to in-memory images. ([#3851](https://github.com/dotnet/machinelearning/pull/3851))
+- Reformatted Ranking samples to width 85 so the horizonal scroll bar does not appear on docs webpage.([#3930](https://github.com/dotnet/machinelearning/pull/3930))
+
+## **Remarks**
+- Github issue count decreased from ~550 to ~380.
```

Comment:
Perhaps mention why it decreased, "we closed x issues dropping the count from .. to .."